Makefile

```
@@ -26,18 +26,24 @@ $(BUILDDIR)/android/tun2socks.aar: $(GOMOBILE)
 	mkdir -p "$(BUILDDIR)/android"
 	$(ANDROID_BUILD_CMD) -o "$@" $(IMPORT_PATH)/outline/tun2socks $(IMPORT_PATH)/outline/shadowsocks
 
-apple: $(BUILDDIR)/apple/Tun2socks.xcframework
+apple_legacy: $(BUILDDIR)/apple_legacy/Tun2socks.xcframework
 
-$(BUILDDIR)/apple/Tun2socks.xcframework: $(GOMOBILE)
+$(BUILDDIR)/apple_legacy/Tun2socks.xcframework: $(GOMOBILE)
```

```suggestion
$(BUILDDIR)/ios/Tun2socks.xcframework: $(GOMOBILE)
```

But move macos to the maccatalyst target.